lib/RegexTools.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var RegexTools = {};
```

Comment:
Should I put `getUniqueMatch` in it's own module? It will be used by another regex in a later PR